generic/openshift/dual-region/procedure/chart-env.sh

```diff
@@ -30,4 +30,5 @@ export AWS_ES_BUCKET_REGION=""
 # The Helm release name used for installing Camunda 8 in both Kubernetes clusters
 export HELM_RELEASE_NAME=camunda
 # renovate: datasource=helm depName=camunda-platform registryUrl=https://helm.camunda.io
-export HELM_CHART_VERSION=11.3.0
+export HELM_CHART_VERSION="0.0.0-snapshot-alpha"
```

Comment:
you probably want to use `0.0.0-snapshot-latest` atm since `alpha` is 8.8 and likely not compatible.